### PR TITLE
Choose Attributes to Display on Detail View

### PR DESCRIPTION
**Pull request by tommartensen**

### CODE REVIEW

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [attributeConfig-template.js](https://github.com/bptlab/argos-frontend/blob/0dc42ed1f424b7899b1c3ce0d88cd413b63f47e5/src/config/attributeConfig/attributeConfig-template.js): Rename this file to "attributeConfig". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/GridView/CardGrid.js [hunk 1]
```diff
@@ -62,11 +63,25 @@ class CardGrid extends ConnectionComponent {
 		const attributeArray = [];
 		childEntity.Attributes.forEach((attribute) => {
 			attributeArray.push(attribute.Name);
-			attributeArray.push(attribute.Value)
+			attributeArray.push(attribute.Value);
 		});
 		return attributeArray;
 	}
 
+	static getNecessaryAttributes(entityType) {
```
Comment (MaximilianV) [inline src/GridView/CardGrid.js]: Why is this not implemented in Utils?

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [attributeConfig-template.js](https://github.com/bptlab/argos-frontend/blob/9188426294343d9dd5ea0178b129c163899fa1ee/src/config/attributeConfig/attributeConfig-template.js): Rename this file to "attributeConfig". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/GridView/CardGrid.js [hunk 2]
```diff
@@ -61,11 +62,25 @@ class CardGrid extends ConnectionComponent {
...
+		const necessaryAttributes = [];
+		const attributeDefinition = attributeConfig[String(entityType.Name)];
+		if(!attributeDefinition) {
+			return null;
+		}
+		Object.entries(attributeDefinition).forEach(([attributeName, isNecessary]) => {
```
Comment (MaximilianV) [inline src/GridView/CardGrid.js]: An _object_ function? Can't you use a function defined on _attributeDefinition_?

**Review by MaximilianV — CHANGES_REQUESTED** [review]

File: src/GridView/CardGrid.js (hunk 1 above)
Comment (tommartensen) [reply src/GridView/CardGrid.js]: Oook

Comment (MaximilianV) [reply src/GridView/CardGrid.js]: 👍

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [attributeConfig-template.js](https://github.com/bptlab/argos-frontend/blob/f3724382c4170e74d3bff086e8c21c9fd284eb23/src/config/attributeConfig/attributeConfig-template.js): Rename this file to "attributeConfig". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS3317)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by MaximilianV — APPROVED** [review]

**Pull request merged**